web/src/components/CanvasBoard.module.css

```diff
@@ -0,0 +1,3 @@
+.canvas {
+    border: 3px solid black;
```

Comment:
🪧 Ban `px` in css

fine as a placerholder, I'd change to `0.2rem` but as this is a placeholder that's up to you

if you're not sure about px vs em ask at the next session